### DISCUSSION TITLE
Flag button missing from new UI

### DISCUSSION THREAD

**Discussion by Hunterius8** [hlky/stable-diffusion-webui#222] [Feature suggestions]

**Comment by hlky** [comment]
feature update needed to change the layout from gradio.Interface to use gradio.Blocks
needs reimplementing, slightly different
if the feature is very important to you then you can do it
example
https://gradio.app/using_flagging/#flagging-with-blocks

`class Flagging(gr.FlaggingCallback)` still exists are provably still works, it's only the button that needs adding

**Comment by Hunterius8** [comment]
I tried and failed, might not be an easy fix. But I also have barely any experience with python, let alone gradio, so I might've just missed an easy solution.

**Comment by hlky** [comment]
it's a useless feature anyway imo

**Comment by altryne** [comment]
What did it do? It was essentially "star" to save the prompts that you think are good separately?

**Comment by hlky** [comment]
Yes, some people would disable save grid or save image or both when doing large batches then flag only the images they want
